### ISSUE TITLE
Story image on story title pages is unusually slow to load.

### ISSUE THREAD

**Issue by TrickQuestion** [knighty/fimfiction-issues#171]
Originally I thought the problem here was that Fimfiction was resizing images dynamically when you go to view a story's title page, but I'm less certain of that now.

The issue is that when you go to a story's title page, the image for the story takes several seconds to load, which changes the appearance of the long description. It now appears that this is a problem even if the story image does not need to be resized at all.

The thumb of the image in other contexts (and even the medium image in the feature box) has no difficulty loading. It's only on the story title page that this appears to be an issue, and it's pretty noticeable.

**Comment by ocalhoun6** [comment]
"The thumb of the image in other contexts (and even the medium image in the feature box) has no difficulty loading."

When browsing a list of stories in 'full' view on mobile, there's a significant lag for each cover image, and they don't begin to load until they're on the screen, which makes it difficult to scroll through. (Scroll to something near the bottom, and then all the images in between begin to load, changing your scroll position.)

**Comment by TrickQuestion** [comment]
I'm not surprised it's worse on mobile, but it's definitely a new issue since the upd8.

This would not be quite as bad if they started putting images on the right-hoof side where they belong, so the formatting doesn't make the long desc hard to read esp. on mobile. (I thought that was planned. Is there a FR on that yet?) But that's a separate issue, even if it would help to mitigate this one.
